Core/admin.py

```diff
@@ -1,9 +1,7 @@
 from django.contrib import admin
 from import_export.admin import ImportExportActionModelAdmin
-from .models import Technology, Member, Designation, Project, ContactUs, \
- Registration, Branch, Gender, Year, ContactInfo, Blog, Event, EmailContent, \
- EmailAttachment
-from .forms import RegistrationForm
+from .models import *
```

Comment:
specify the names of functions imported as it was before

Reply:
@bvishal8510 sir it will automatically import all the functions and reduces code